apps/wallet/src/ui/app/components/ledger/useDeriveLedgerAccounts.ts

```diff
@@ -4,27 +4,34 @@
 import { Ed25519PublicKey } from '@mysten/sui.js';
 import { useEffect, useState } from 'react';
 
-import { type LedgerAccount } from './LedgerAccountItem';
 import { useSuiLedgerClient } from './SuiLedgerClientProvider';
+import { AccountType } from '_src/background/keyring/Account';
+import { type SerializedLedgerAccount } from '_src/background/keyring/LedgerAccount';
 
 import type SuiLedgerClient from '@mysten/ledgerjs-hw-app-sui';
 
+export type SelectableLedgerAccount = SerializedLedgerAccount & {
+    isSelected: boolean;
+};
+
 type UseDeriveLedgerAccountOptions = {
     numAccountsToDerive: number;
     onError: (error: unknown) => void;
 };
 
 type UseDeriveLedgerAccountResult = [
```

Comment:
I generally just return as const to avoid this

Reply:
o solid tip, thanks!